src/main/java/com/mraof/minestuck/entity/item/GristEntity.java

```diff
@@ -199,27 +243,67 @@ protected void readAdditionalSaveData(CompoundTag compound)
 			this.gristType = GristType.read(compound, "Type");
 	}
 	
+	public int shaderAlpha = 0;
```

Comment:
Member variables should go at the top of the class, like where you have "animation"

Reply:
understood